### ISSUE TITLE
Honda: check regen paddle

### ISSUE THREAD

**Issue by plygrnd** [commaai/opendbc#1100]
## TL;DR

Honda Insight's brake regen adjustment paddles cause OP to throw a CruiseDisabled error instead of canceling cruise control.

## More Info

The Honda Insight has adjustable regenerative braking. This is controlled by paddles on the steering wheel. Hitting either paddle when cruise is engaged will disengage cruise. 

Disengaging cruise control in this fashion causes OP to trigger a "cruise not engaged" message and sound the alarm, instead of gracefully exiting cruise control via the "Cancel" button. This is governed in the Honda interface, [L332](https://github.com/commaai/openpilot/blob/a2bb41e0ec5fd96763e9a01a7116efda059de6e6/selfdrive/car/honda/interface.py#L332).  

It appears that the car sends a generic "Cruise Control disabled" message; there's a lot of junk in Cabana for this trip which isn't mapped to any signals, but one of those could be the issue.

### Desired Outcome

When using the steering wheel paddles to disable cruise control, OP should treat this as a standard disablement instead of an emergency.

### Which car does this affect?

Honda Insight EX 2019

### Provide a route where the issue occurs

8958942b212e8cd9|2023-11-30--08-18-06--10

### openpilot version

On `release3`, latest release.

**Comment by vanillagorillaa** [comment]
Hi @plygrnd I think I have resolved this issue. If you are on the comma discord do you mind sending me a DM? I have the same username

**Comment by jyoung8607** [comment]
For benefit of anyone that's experiencing this bug, looks like @vanillagorillaa has a good start on the issue in #2229 but we're blocked looking for an Insight, Clarity, or other test user who has regen paddles that are currently causing problems with openpilot disengagement. If you're experiencing this issue, please share a link to a route in this issue.

**Comment by vanillagorillaa** [comment]
FWIW, i'm no longer working/contributing on/to this issue. Good luck to however decides to pick it up

**Comment by jyoung8607** [comment]
Potentially connected, the Insight DBC has a weird difference in its STANDSTILL message which I suspect is a bug. If it's a bug, it might falsely trigger `accFaulted` under openpilot longitudinal. This could explain difficulties fixing the regen paddle.

**Comment by jyoung8607** [comment]
Notes:

* The stock ACC case will be resolved by #2624.
* The openpilot longitudinal case will require some safety code refactoring.
* Back out commaai/openpilot#35929 before closing.

**Comment by mvl-boston** [comment]
A CRV Hybrid driver noticed a bug with 2624:

"usually hitting regen paddles disengages OP and hitting resume cruise will engage it again. However, I got a "pedal pressed" error when I hit resume. In the beginning, everything works fine like it supposed to but after I try it a few times i got that error message. I had to hit set cruise to engage OP again. I think the culprit could be the right paddle. I tested it it and that seemed to be the case."

0a84f6dd8372aef6/00000000--2b607c819d

**Comment by jyoung8607** [comment]
> A CRV Hybrid driver noticed a bug with 2624:

#2643
